docs/esp8266/quickref.rst

```diff
@@ -183,6 +183,19 @@ The SPI driver is implemented in software and works on all pins::
     spi.write_readinto(b'1234', buf) # write to MOSI and read from MISO into the buffer
     spi.write_readinto(buf, buf) # write buf to MOSI and read MISO back into buf
 
+
+Hardware SPI
+------------
+
+The hardware SPI is faster (up to 80Mhz), but only works on following pins:
+``MISO``=12, ``MOSI``=13, ``SCK``=14. It has the same methods as SPI, except
```

Comment:
It's unclear what "12", etc. mean here. Physical chip pin number? GPIO number? (NodeMCU peoplew will think it's nodemcu logical number, etc.) It should be "GPIO12", etc. to disambiguate.